src/st_widgets.c

```diff
@@ -829,13 +843,28 @@ static void UpdateMonSec(sbe_widget_t *widget)
     StatsFormatFunc(item_str, sizeof(item_str), fullitemcount, totalitems);
     StatsFormatFunc(secret_str, sizeof(secret_str), fullsecretcount, totalsecret);
 
-    M_snprintf(string, sizeof(string),
-        RED_S "K \x1b%c%s " RED_S "I \x1b%c%s " RED_S "S \x1b%c%s",
-        killcolor, kill_str,
-        itemcolor, item_str,
-        secretcolor, secret_str);
-
-    ST_AddLine(widget, string);
+    if (!widget->vertical)
+    {
+        static char string[120];
```

Comment:
I think we can make these buffers a lot smaller, since we are not tracking individual multiplayer stats anymore.

Reply:
By the way, why don't we track multiplayer stats anymore? I can bring them back.

Reply:
Because we followed suit with dsda-doom (which dropped multiplayer support) when we implemented "smart" kill counts.

Reply:
> "smart" kill counts.

Oh, I didn't know.

> I think we can make these buffers a lot smaller

Done in https://github.com/fabiangreffrath/woof/pull/2168/commits/430ebdf329caa5610f0ed676f66ccab30a1cf2ea

Reply:
> Because we followed suit with dsda-doom (which dropped multiplayer support) when we implemented "smart" kill counts.

However, it may still make sense to have a stats widget adjusted for co-op multiplayer, just as we already have a separate counter widget for deathmatch. We do actively support multiplayer games after all.

Reply:
> However, it may still make sense to have a stats widget adjusted for co-op multiplayer.

I don't know how smart counts work yet 😄  Maybe later, nobody complains about it yet.